Selenium_Basics/EpamNavigationTests2.cs

```diff
@@ -0,0 +1,109 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Selenium_Basics;
+
+public class EpamNavigationTests2
+{
+    private IWebDriver driver { get; set; }
+
+    [SetUp]
+    public void SetupBrowser()
+    {
+        var options = new ChromeOptions();
+        options.BinaryLocation = "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome";
+        driver = new ChromeDriver(options);
+    }
+    
+    [TearDown]
+    public void TearDown()
+    {
+        driver.Quit();
+    }
+    
+    [Test]
+    public void CheckThatListOfCountriesContainsAmerEMEAandAPAC()
+    {
+        var epamUrl = "https://www.epam.com/";
+
+        driver.Manage().Window.Maximize();
```

Comment:
we can put those lines in setup as they presented in all tests
driver.Manage().Window.Maximize();
        driver.Navigate().GoToUrl(epamUrl);
and variable var epamUrl = "https://www.epam.com/"; could be placed in constants